### PR TITLE
feat: Add main search page

### PR DESCRIPTION
**Pull request by andres-acc**
## Summary
* Add main search page
* Add colors
* Add bacground images

## Preview
Screen Shot 2023-03-13 at 6 23 31 PM

## Trello Card
https://trello.com/c/smrt5zJX

### CODE REVIEW

File: src/app/components/result-list/character-result-box/character-result-box.component.ts [hunk 1]
```diff
@@ -2,6 +2,7 @@ import { Component, Input } from '@angular/core';
 import { Gender } from 'src/app/helpers/enums/gender.enum';
 import { BasicCharacter } from 'src/app/interfaces/character.interface';
 import { Species } from '../../../helpers/enums/species.enum';
+import { Status } from '../../../helpers/enums/status.enum';
```
Comment (YgRamos9) [inline src/app/components/result-list/character-result-box/character-result-box.component.ts:5]: - use absolute path please

File: src/app/pages/search-character/search-character.component.scss [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  }
+
+  .input {
+    margin: -50px auto;
+    min-width: 585px;
+    width: 50%;
+
+    &::before {
+      content: '';
+      background: url('../../../assets/images/main-background.png');
```
Comment (YgRamos9) [inline src/app/pages/search-character/search-character.component.scss:39]: - you could also use

```suggestion
      background: url('/assets/images/main-background.png');
```

**Review by YgRamos9 — APPROVED** [review]

File: src/app/pages/search-character/search-character.component.scss [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+      content: '';
+      background: url('/assets/images/main-background.png');
+      background-repeat: no-repeat;
+      background-size: cover;
+      bottom: 0px;
+      position: absolute;
+      left: 0px;
+      opacity: 0.15;
+      right: 0px;
+      top: 40vh;
```
Comment (miguelhenaoa) [inline src/app/pages/search-character/search-character.component.scss:48]: sort

File: src/app/pages/search-character/search-character.module.ts [hunk 4]
```diff
@@ -2,10 +2,15 @@ import { NgModule } from '@angular/core';
 import { CommonModule } from '@angular/common';
 
 import { SearchCharacterComponent } from './search-character.component';
+import { BigInputModule } from 'src/app/components/big-input/big-input.module';
```
Comment (miguelhenaoa) [inline src/app/pages/search-character/search-character.module.ts:5]: Move above

**Review by miguelhenaoa — CHANGES_REQUESTED** [review]

**Review by miguelhenaoa — APPROVED** [review]

**Pull request merged**